rustls/tests/api.rs

```diff
@@ -5133,6 +5133,39 @@ fn test_server_picks_dhe_group_when_clienthello_has_no_dhe_group_in_groups_ext()
     assert!(server.process_new_packets().is_ok());
 }
 
+#[cfg(feature = "tls12")]
+#[test]
+fn test_server_picks_dhe_group_when_clienthello_has_no_groups_ext() {
```

Comment:
Check my understanding, this only happens when client and server agree with using a DHE ciphersuite?

Reply:
Yes that's correct!